server/lib/packets/topic.go

```diff
@@ -0,0 +1,43 @@
+package packets
+
+import (
+	"log"
+
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/models"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/lib/models (can't find import: github.com/umegaya/channer/server/lib/models)